share/smack/doctor.py

```diff
@@ -74,35 +74,35 @@ def check_headers(prefix):
             check("%s contains %s" % (header_file, content), content in open(header_file).read())
 
 def main():
-    global ARGS
-    global COUNT
+    global smack_doctor_args
+    global issue_count
 
     parser = argparse.ArgumentParser(description='Diagnose SMACK configuration issues.')
     parser.add_argument('-q', '--quiet', dest='quiet', action="store_true", default=False,
                         help='only show failed diagnostics')
     parser.add_argument('--prefix', metavar='P', dest='prefix', type=str, default='',
                         help='point to the installation prefix')
-    ARGS = parser.parse_args()
-    COUNT = 0
+    smack_doctor_args = parser.parse_args()
+    issue_count = 0
```

Comment:
Why would you replace more descriptive names (like `issue_count`) with less descriptive names (like just `COUNT`)?
There are several global names that you truncated like that to make them harder to understand.

Reply:
I think your comments refer to old changes. The upper case names are removed.